### PR TITLE
fix german translation

### LINKED ISSUE #381: German translation has errors and typos
There are some wrong translations and typos in the po file for the German language.

### PR DESCRIPTION
**Pull request by d-s-e**
Closes #381.

### CODE REVIEW

**Review by uhurusurfa — DISMISSED** [review]
🎸 - vielen dank!

**Comment by frlan** [comment]
Yepp, looks better like this. Please merge anyone ;)

File: account/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -216,7 +216,7 @@ msgstr "Sie haben {email} bestätigt."
 #: account/views.py:463
 #, python-brace-format
 msgid "Email confirmation for {email} has expired."
-msgstr ""
+msgstr "Die E-Mail-Bestätigung ist abgelaufen."
```
Comment (blag) [inline account/locale/de/LC_MESSAGES/django.po:219]: Should this include the `{email}`?

Comment (d-s-e) [reply account/locale/de/LC_MESSAGES/django.po]: You're right, it should. Fixed.

**Pull request merged**